tool_collections/cufflinks/cuffdiff/cuffdiff_wrapper.xml

```diff
@@ -186,6 +187,74 @@
         </conditional>
     </inputs>
     <outputs>
+        <!-- Optional read group datasets. -->
+        <data format="tabular" name="isoforms_read_group" label="${tool.name} on ${on_string}: isoforms read group tracking" from_work_dir="isoforms.read_group_tracking" >
```

Comment:
One could do that but I would recommend strongly against - data collections should be homogeneous. These tabular files each have a different purpose - they are not just different samples for instance.